nat-lab/tests/test_dns.py

```diff
@@ -860,3 +860,111 @@ async def test_dns_nickname() -> None:
         )
         for ip in beta.ip_addresses:
             assert ip in beta_response.get_stdout()
+
+
+@pytest.mark.asyncio
+async def test_dns_change_nickname() -> None:
```

Comment:
I'm not sure how this test relates to the restriction of nat-lab containers 🤔

Reply:
you're right it's unrelated, I should have mentioned it

Reply:
If that is the case, can you extract it to separate review?

Reply:
✅